mrLoadRet/Plugin/GLM_v2/newGlmAnalysis/glmPlot_edit.m

```diff
@@ -0,0 +1,852 @@
+function glmPlot_edit(thisView,overlayNum,scanNum,x,y,s,roi)
```

Comment:
what is the difference between glmPlot_edit and the original glmPlot? Are there some new features that could benefit people who use glmPlot, and can these features be integrated into glmPlot instead?
Otherwise, if it was just for convenience to do soemthing quickly I'd rather not add this function to the main mrTools repository. 
Instead you could set the "interrogatorPaths" preferrence to add an alternative folder where you can add your own interrogator function (using mrSetPref)